mobile/src/reducers/organization/organization.reducer.ts

```diff
@@ -4,20 +4,32 @@ import { combineReducers } from 'redux';
 import { OrganizationState } from './organization.reducer';
 import { departmentsReducer } from './departments.reducer';
 import { employeesReducer, EmployeesStore } from './employees.reducer';
-import { loadDepartmentsEpic$, loadChiefsEpic$, loadDepartmentsFinishedEpic$, loadEmployeesForDepartmentEpic$ } from './organization.epics';
+import {
+    loadDepartmentsEpic$, loadChiefsEpic$, loadDepartmentsFinishedEpic$, loadEmployeesForDepartmentEpic$,
+    loadUserEpic$, 
+    loadEmployeeEpic$,
+    loadUserFinishedEpic$} from './organization.epics';
+import { userReducer, UserState } from './user.reducer';
+import { Employee } from './employee.model';
+import { User } from './user.model';
 
 export interface OrganizationState {
     departments: Department[];
     employees: EmployeesStore;
+    user: UserState;
 }
 
 export const organizationEpics = combineEpics(
     loadDepartmentsEpic$ as any,
     loadChiefsEpic$ as any,
     loadDepartmentsFinishedEpic$ as any,
-    loadEmployeesForDepartmentEpic$ as any);
+    loadEmployeesForDepartmentEpic$ as any,
+    loadUserEpic$ as any,
+    loadUserFinishedEpic$ as any,
+    loadEmployeeEpic$ as any);
 
 export const organizationReducer = combineReducers<OrganizationState>({
     departments: departmentsReducer,
-    employees: employeesReducer
+    employees: employeesReducer,
+    user: userReducer
```

Comment:
I do not think that user belongs under organization. Should it be in a folder of its own? We'll have to support some settings later, for example.

Reply:
Fixed